index.html

```diff
@@ -339,42 +339,43 @@ <h2>JSON Schema Specifications</h2>
                 <tr>
                    <th style="white-space: nowrap">JSON Schema Specification</th>
                    <th>Date of Publication</th>
+                   <th>$schema URI</th>
                    <th>References</th>
                 </tr>
              </thead>
              <tbody>
                 <tr>
-                   <td><a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-01">draft-bhutton-json-schema-01</a></td>
+                   <td>[[JSON-SCHEMA-2020-12]]</td>
                    <td>10 June 2022</td>
+                   <td><a href="https://json-schema.org/draft/2020-12/schema">https://json-schema.org/draft/2020-12/schema</a></td>
                    <td>
                       • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-01">JSON Schema: A Media Type for Describing JSON Documents</a></br>
                       • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-01">JSON Schema Validation: A Vocabulary for Structural Validation of JSON</a></br>
                       • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-relative-json-pointer-00">Relative JSON Pointers</a>
                    </td>
                 </tr>
                 <tr>
-                   <td><a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">Draft 2020-12, draft-bhutton-json-schema-00</a></td>
-                   <td>11 June 2021</td>
+                   <td>[[JSON-SCHEMA-2019-09]]</td>
+                   <td>19 March 2020</td>
+                   <td><a href="https://json-schema.org/draft/2019-09/schema">https://json-schema.org/draft/2019-09/schema</a></td>
                    <td>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">JSON Schema: A Media Type for Describing JSON Documents</a></br>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00">JSON Schema Validation: A Vocabulary for Structural Validation of JSON</a></br>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-relative-json-pointer-00">Relative JSON Pointers</a>
+                      • <a href="https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-02">JSON Schema: A Media Type for Describing JSON Documents</a></br>
```

Comment:
not sure about all these IETF refs, prefer to not mix IETF and OpenJS refs for the same standard, I suggest not referring to IETF drafts at all.